.github/actions/install-oppia-dependencies/action.yml

```diff
@@ -0,0 +1,24 @@
+name: 'Install Dependencies'
+description: 'Installs wheel and third-party python libs.'
+runs:
+  using: 'composite'
+  steps:
+    - name: Setup python by installing wheel
+      run: pip install wheel==0.35.0
+      shell: bash
+    - name: Check Yarn Cache
+      run: |
+        pwd
+        cd ..
+        pwd
+        echo 'cache-folder "yarn_cache"' > .yarnrc
```

Comment:
Why are we doing it manually here? [I see the.yarnrc file is already changed in this PR] (But I liked this approach as well.)

Reply:
Notice the `cd ..`. We are moving out of the `oppia` folder, so this line creates a *new* `.yarnrc` file. This is because the Oppia `.yarnrc` file specifies a path to the yarn binary. Since we haven't downloaded that binary yet, we have to move to a different folder in order to use `yarn` commands to check the cache.